addons/udes_priorities/models/priority.py

```diff
@@ -37,6 +45,12 @@ def _default_sequence(self):
         string="Stock Transfer Types",
         help="The stock picking types this priority is visible on. No picking types means all picking types",
     )
+    affected_pickings = fields.Many2many(
+        "stock.picking",
+        string="Affected Stock Transfers",
+        compute="_get_default_affected_pickings",
```

Comment:
Suggest to rename the compute method to '_compute_affected_pickings`, that is the usual convention for the methods of computed fields.

Reply:
Just realised there is a typo in `_number_of_oustanding_pickings`, we might want to fix this too :)